File: if else/lastdigit.c
/*write a program to display the last digit of a num*/

#include <stdio.h>

int main() {
    int num, lastDigit;

    printf("Enter any number: ");
    scanf("%d", &num);

    lastDigit = num % 10;   // gives last digit

    printf("Last digit is: %d", lastDigit);

    return 0;
}

File: if else/lastdigit.java
/*#write a program to display the last digit of a num*/

import java.util.Scanner;

class LastDigit {
    public static void main(String args[]) {

        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number:");
        int num = sc.nextInt();

        int lastDigit = (num % 10);
        System.out.println("The last digit of " + num + " is: " + lastDigit);
    }
}



